arch/src/aarch64/fdt.rs

```diff
@@ -81,6 +89,119 @@ pub enum Error {
 }
 type Result<T> = result::Result<T, Error>;
 
+/// L1 data cache
+const L1_D_CACHE: u32 = 0;
+/// L1 instrcution cache
+const L1_I_CACHE: u32 = 1;
+/// L2 cache
+const L2_CACHE: u32 = 2;
+/// L3 cache
+const L3_CACHE: u32 = 3;
+
+///let mut file_directory = "/sys/devices/system/cpu/cpu0/cache/index0/size";
+pub fn get_cache_size(cache_level: u32) -> u32 {
```

Comment:
Ditto

Reply:
It is not commented out code, just annotation. I will modify it soon.